apps/login-register/src/app/register/register.component.ts

```diff
@@ -34,5 +60,34 @@ export class RegisterComponent implements OnInit {
       email: ['', [Validators.required, Validators.email]],
       password: ['', [Validators.required, Validators.minLength(8)]],
     });
+
+    const firstName = this.registerForm.get('firstName');
```

Comment:
extract this into a separate method and use it in the onInit

Reply:
same